packages/api-client/src/TamanuApi.js

```diff
@@ -80,7 +80,7 @@ export class TamanuApi {
     this.setToken(token);
 
     const { user, ability } = await this.fetchUserData(permissions);
-    return { user, token, localisation, server, availableFacilities, ability, role };
+    return { user, token, localisation, server, availableFacilities, ability, role, permissions };
```

Comment:
This is a related bug but was not the main issue. Downstream we use this permissions list to build the user ability.